static/js/main.js

```diff
@@ -62,9 +62,9 @@ var header = {
         var password = $("#password").val();
         var verify_password = $("#verify_password").val();
 
-        if (!$("#user_id") || user_id.length == 0) {
+        if (!$("#user_id") || user_id.length == 0 || !validateEmail(user_id)) {
```

Comment:
Can you add this validation to forgot password form as well?

Reply:
Do you want to make separate function for forgot password in main.js and use this method in js file or do you want to implement another checking method for python code? Cause for now, forgot password is under views.py python file.